Resources/public/templates/studiopluspresentation.hbt

```diff
@@ -1,22 +1 @@
-<p>You don't have eZ Studio Plus Installed :(</p>
-
-<h1>Learn more about eZ Studio Plus</h1>
-
-<p>
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin facilisis tristique felis, in euismod
-    ipsum sollicitudin vitae. Morbi pellentesque ut purus non placerat. Vivamus sollicitudin condimentum
-    nunc, eget euismod dolor. Etiam ultricies sit amet dui vel sagittis. Maecenas vitae fermentum urna.
-    Sed aliquam lacinia ante, ac convallis justo pulvinar ac. Nunc pellentesque lobortis diam, nec
-    placerat sapien pulvinar ac. Nullam euismod mattis dictum. Suspendisse consectetur, nisl vel
-    porttitor bibendum, eros nisi placerat est, ut mattis magna lorem ac dolor. Aliquam viverra sit
-    amet purus eu tincidunt.
-</p>
-
-<p>
-Quisque ut tincidunt elit. Pellentesque nulla felis, vestibulum non eros vitae, lacinia luctus odio.
-    Etiam dapibus metus eget bibendum mattis. Donec lectus risus, hendrerit non condimentum et,
-    venenatis nec neque. Etiam sed varius arcu. Nulla ullamcorper luctus augue, blandit tempor neque
-    lobortis sit amet. Mauris ac eleifend augue. Etiam condimentum auctor ex, nec vehicula nunc aliquam
-    et. Quisque sagittis sed mi maximus rutrum. Proin eu enim non neque viverra iaculis. In hac habitasse
-    platea dictumst. Aenean euismod nunc urna, id iaculis nisl gravida vel.
-</p>
+<iframe class="ez-studiopluspresentation-content" src="about:blank"></iframe>
```

Comment:
@andrerom we can add the `seamless` attribute but no browser is supporting it http://caniuse.com/#search=seamless